src/kili/entrypoints/queries/label/__init__.py

```diff
@@ -642,7 +642,7 @@ def export_labels(
             project_id: Identifier of the project.
             filename: Relative or full path of the archive that will contain
                 the exported data.
-            fmt: Format of the exported labels.
+            fmt: Format of the exported labels. One of "raw", "kili", "yolo_v4", "yolo_v5", "yolo_v7", "coco", "pascal_voc".
```

Comment:
it is already highlighted through the `LabelFormat` type: 
<img width="850" alt="image" src="https://user-images.githubusercontent.com/1255753/230333159-6f95954a-3e3b-437b-9d99-59a327b74098.png">

Reply:
![image](https://user-images.githubusercontent.com/15847892/230334418-82c63faf-55b2-44ae-995e-a00650c398fc.png)

it doesn't show in the vscode showHover

Reply:
<img width="984" alt="image" src="https://user-images.githubusercontent.com/1255753/230345249-6c0bad11-da9f-4941-9538-4f72f3c54c5e.png">

Reply:
it shows the options when you start writing. But I agree that you have to know the trick.

Reply:
I removed it